my-app/src/Loading.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Child Class 5 - handles loading message
+ * Stateful Component Class 
+ * 
+ * Ideally we want it to run when a search is called, but not sure how to do this
```

Comment:
You could try setting state in the `App` component then pass it down as a prop:

```
setState({
   searchInProgress: true
})
```

The initial value for `searchInProgress` could be set to `false` in the `App` constructor.